assets/src/components/account/AccountOverview.tsx

```diff
@@ -118,19 +127,22 @@ class AccountOverview extends React.Component<Props, State> {
     company_name?: string;
     time_zone?: string;
     working_hours?: Array<WorkingHours>;
+    widget_settings?: UpdateWidgetSettings;
```

Comment:
Phoenix notes it'll create an extra association if the nested entity doesn't possess an ID, so I added a lil guard here in the form of an UpdateType... not sure how much we're concerned about it really, more just thinking through how to protect from something like this in general.

I think a type is elegant, so is probably an update-specific changeset on the BE (tho that will show an error to the user, not amazing really). I dunno, this seems pretty good I guess. 

I definitely don't intend to start typing each type of CRUD for no reason, this just seemed like a reasonable place to do it

Reply:
hmm I wonder if it would be simpler to just use the `API.updateWidgetSettings` to handle the `widget_settings` update (since that makes sure we only update the correct `widget_settings` record based on the auth token)?

then we could just handle calling that in the `handleUpdateHideWidgetOutsideWorkingHours` without worrying about any of the other account-related fields 🤷 what do you think?